app/components/search/SearchResults/SearchResults.tsx

```diff
@@ -64,6 +64,13 @@ const SearchResults = ({
     );
   };
 
+  function handleAction(searchMapAction: SearchMapActions) {
+    switch (searchMapAction) {
+      case SearchMapActions.SearchThisArea:
+        return refinePagination(0);
+    }
+  }
```

Comment:
**note:** Generally I wouldn't create switch for a single case but I'm going a little overboard here to put more intent into the code that describes _reducer_-like behavior over a non-discreet set of possible actions. Even if that number of possible actions is currently 1, this should clue developers that this component is acting as a manager for user interaction with the map.